app/assets/stylesheets/custom.css.scss

```diff
@@ -130,3 +130,17 @@ li { list-style: none; }
 .values {
   color: $site-white;
 }
+
+.bw {
+  -webkit-filter: grayscale(100%);
+  -webkit-transition: all .5s ease;
+     -moz-transition: all .5s ease;
+       -o-transition: all .5s ease;
+      -ms-transition: all .5s ease;
```

Comment:
Line should be indented 2 spaces, but was indented 6 spaces<br>`.5` should be written with a leading zero as `0.5`<br>Avoid vendor prefixes.